app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -609,24 +609,29 @@ public static String activeSlopeArrow() {
         return slopeToArrowSymbol(slope);
     }
 
-    public static String slopeToArrowSymbol(double slope) {
+    public static int slopeToArrowIndex(double slope) {
         if (slope <= (-3.5)) {
-            return "\u21ca";// ⇊
+            return 0;// ⇊
         } else if (slope <= (-2)) {
-            return "\u2193"; // ↓
+            return 1; // ↓
         } else if (slope <= (-1)) {
-            return "\u2198"; // ↘
+            return 2; // ↘
         } else if (slope <= (1)) {
-            return "\u2192"; // →
+            return 3; // →
         } else if (slope <= (2)) {
-            return "\u2197"; // ↗
+            return 4; // ↗
         } else if (slope <= (3.5)) {
-            return "\u2191"; // ↑
+            return 5; // ↑
         } else {
-            return "\u21c8"; // ⇈
+            return 6; // ⇈
         }
     }
 
+    public static String slopeToArrowSymbol(double slope) {
```

Comment:
Can you explain the reason for refactoring this method?

Reply:
The up-right and down-right arrow symbols from the font are visually very different than the normal up, down, and right arrows. I did this refactoring in preparation to add some pure images for the arrows, and was going to refactor the gear widget to use them. Never got around to finishing it, but I intend to in the near future.

I've got the code in a local branch, I'll remove it for now so it's not confusing.